plugins/woocommerce/changelog/53453-dev-51473-switch-to-live-account

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: dev
+
+Switch from test-drive to live account from the settings page
```

Comment:
It's not clear which account we are referring to here. Also, IIRC we still use the term "sandbox" externally to refer to test-drive mode.

```suggestion
Allow to switch from sandbox to live WooPayments account from the Payment settings page.
```